src/containers/Login.js

```diff
@@ -1,81 +1,98 @@
-import React from 'react';
-import { Link } from 'react-router-dom';
+import React from "react";
+import { Link } from "react-router-dom";
 
-// Local Imports
-import apiClient from '../lib/apiClient';
+import apiClient from "../lib/apiClient";
 
-// Styles
-import '../styles/LoginSignup.css';
+import { invalidEmail } from "../helpers/stringHelper";
+
+import "../styles/LoginSignup.css";
```

Comment:
Sometimes we have `//styles` to separate our imports. I don't mind either way, but this file doesn't have that. We should either add that here or remove it from the other files to be consistent.

Reply:
I think it's self-explanatory this is a styles file, so I will remove.